### PR TITLE
NO JIRA. Changed Deposit license to Deposit agreement.

### PR DESCRIPTION
**Pull request by janvanmansum**
NO JIRA

#### When applied it will
* Update the terminology with respect to the depositor agreement.

### CODE REVIEW

File: src/main/typescript/lib/metadata/Metadata.ts [hunk 1]
```diff
@@ -178,7 +178,7 @@ export const metadataConverter: (input: any, dropDowns: DropdownLists) => Deposi
         // privacy sensitive data
         privacySensitiveDataPresent: privacySensitiveDataPresent,
 
-        // deposit license
+        // depositor agreement
         acceptLicenseAgreement: input.acceptLicenseAgreement || false,
```
Comment (rvanheest) [inline src/main/typescript/lib/metadata/Metadata.ts]: are you sure you don't want to change the fieldname in the data model as well? That would require changes in several files as well...

**Comment by lindareijnhoudt** [comment]
Official term chosen is `DANS Deposit Agreement`, in the heading of the card we can use `Deposit Agreement`

File: src/main/typescript/lib/metadata/Metadata.ts (hunk 1 above)
Comment (janvanmansum) [reply src/main/typescript/lib/metadata/Metadata.ts]: I'll create a separate issue for it, as this requires changes in both API and UI.

Comment (rvanheest) [reply src/main/typescript/lib/metadata/Metadata.ts]: no, it doesn't. There is a mapping from external model to internal model, so changing the internal model's fields doesn't require changes to the external model and/or API

**Pull request merged**